.github/wiki/FirstPullRequest.md

```diff
@@ -0,0 +1,120 @@
+
+For example, we use [Prettier](https://prettier.io/) to format our JavaScript code. This saves you time and energy as you can let Prettier fix up any formatting issues automatically through its editor integrations, or by manually running `npm run prettier`. We also use a linter to catch styling issues that may exist in your code. You can check the status of your code styling by simply running `npm run lint`.
+
+To learn more about coding conventions, refer to the [Style Guide](#style-guide).
+
+## Chapter III: Proposing your Changes
+
+### 1. Commit your changes
+
+Make sure to add your changes to version control using `git`:
+
+```bash
+git add <filename>
+git commit -m <message>
+```
+
+You can use a short descriptive sentence as your commit message.
+
+> Worried about writing good git commit messages? Do not fret. Later, when your pull request is merged, all your commits will be squashed into a single commit. It is your pull request description which will be used to populate the message for this squashed commit.
+
+This guide covers just enough to help you along with your first contribution. GitHub has several resources to help you get started with git:
+
+- [Using Git](https://help.github.com/en/categories/using-git)
+- [The GitHub Flow](https://guides.github.com/introduction/flow/)
+
+
+### 2. Push your changes to GitHub
+
+Once your changes have been commited to version control, you can push them to GitHub.
+
+```bash
+git push fork <my_feature_branch>
+```
+
+If all goes well, you will see a message encouraging you to open a pull request:
+
+```bash
+remote:
+remote: Create a pull request for 'your_feature_branch' on GitHub by visiting:
+remote:      https://github.com/your_username/react-native/pull/new/your_feature_branch
+remote:
+```
+
+Visit the provided URL to proceed to the next step.
+
+### 3. Create your pull request
+
+You are almost there! The next step is to fill out the pull request. Use a descriptive title that is not too long. Then, make sure to fill out all of the fields provided by the default pull request template:
+
+**Summary:** Use this field to provide your motivation for sending this pull request. What are you fixing?
+
+**Changelog:** Help release maintainers write release notes by providing a short description of what will be changed should the pull request get merged.
+
+**Test Plan:** Let reviewers know how you tested your changes. Did you consider any edge cases? Which steps did you follow to make sure your changes have the desired effect? See [What is a Test Plan?][test-plan] to learn more.
+
+[test-plan]: https://medium.com/@martinkonicek/what-is-a-test-plan-8bfc840ec171
+
+### 4. Review and address feedback
+
+Keep an eye on any comments and review feedback left on your pull request on GitHub. Maintainers will do their best to provide constructive, actionable feedback to help get your changes ready to be merged into the core React Native repository.
```

Comment:
```suggestion
Keep an eye on any comments and review feedback left on your pull request on GitHub. Maintainers will do their best to provide constructive, actionable feedback to help get your changes ready to be merged into the core React Native repository.

### 5. Merged!

After your change has been approved and merged by a maintainer, you've successfully added your first change to React Native!
```

Just a suggestion for a final section to close on, my wording here could be improved